File: CourseProjects/CourseProjects/JavaEclipseProjects/JavaLanguageExamples/src/demos/generics/collections/Employee.java
package demos.generics.collections;

class Employee {
	public Employee(String id, String name, double salary) {
		 super();
		 this.id = id;
		 this.name=name;
		 this.salary = salary;
	}
	public String toString() {
		  return name + " with ID " + id + " and salary " + salary;
	}
	private String id;
	private String name;
	private double salary;
}

File: CourseProjects/CourseProjects/VisualStudioProjects/Languages/VS2008 Features/InitializationSyntaxCS/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InitializationSyntaxCS {
    public class Employee {
        public string Name {
            get { return name; }
            set { name = value; }
        }
        public int Age {
            get { return age;  }
            set { age = value; }
        }
        public double Salary {
            get { return salary;  }
            set { salary = value; }
        }
        public override string ToString() {
            return string.Format("{0} of age {1} earning {2}", name, age, salary);
        }
        private string name;
        private int age;
        private double salary;
    }
}
